superbench/cli/_handler.py

```diff
@@ -72,51 +131,99 @@ def deploy_command_handler(
+    host_file = check_argument_file('host_file', host_file)
+    private_key = check_argument_file('private_key', private_key)
 
-    raise NotImplementedError
+    # Docker config
+    docker_config = OmegaConf.create(
+        {
+            'image': docker_image,
+            'username': docker_username,
+            'password': docker_password,
+            'registry': split_docker_domain(docker_image)[0],
+        }
+    )
+    # Ansible config
+    ansible_config = OmegaConf.create(
+        {
+            'host_file': host_file,
+            'host_list': host_list,
+            'host_username': host_username,
+            'host_password': host_password,
+            'private_key': private_key,
+        }
+    )
 
+    sb_config, output_dir = process_config_arguments(config_file, config_override)
 
-def exec_command_handler(
-    docker_image=None, docker_username=None, docker_password=None, config_file=None, config_override=None
-):
+    return docker_config, ansible_config, sb_config, output_dir
+
+
+def version_command_handler():
+    """Print the current SuperBench tool version.
+
+    Returns:
+        str: current SuperBench tool version.
+    """
+    return superbench.__version__
+
+
+def exec_command_handler(config_file=None, config_override=None):
     """Run the SuperBench benchmarks locally.
 
     Args:
-        docker_image (str, optional): Docker image URI.
-        docker_username (str, optional): Docker registry username if authentication is needed. Defaults to None.
-        docker_password (str, optional): Docker registry password if authentication is needed. Defaults to None.
         config_file (str, optional): Path to SuperBench config file. Defaults to None.
         config_override (str, optional): Extra arguments to override config_file,
             following [Hydra syntax](https://hydra.cc/docs/advanced/override_grammar/basic). Defaults to None.
 
     Raises:
         CLIError: If input arguments are invalid.
     """
-    if bool(docker_username) != bool(docker_password):
-        raise CLIError('Must specify both docker_username and docker_password if authentication is needed.')
-    check_argument_file('config_file', config_file)
+    sb_config, output_dir = process_config_arguments(**locals())
 
-    # Docker config
-    docker_config = OmegaConf.create()
-    for key in ['image', 'username', 'password']:
-        docker_config[key] = eval('docker_{}'.format(key))
-    # SuperBench config
-    sb_config = get_sb_config(config_file)
-    if config_override:
-        sb_config_from_override = OmegaConf.from_dotlist(config_override)
-        sb_config = OmegaConf.merge(sb_config, sb_config_from_override)
+    executor = SuperBenchExecutor(sb_config, output_dir)
+    executor.exec()
 
-    # Create output directory
-    output_dir = create_output_dir()
 
-    executor = SuperBenchExecutor(sb_config, docker_config, output_dir)
-    executor.exec()
+def deploy_command_handler(
```

Comment:
Do we have the requirement in the future to specify container name? or we can confirm that every time create a new container?